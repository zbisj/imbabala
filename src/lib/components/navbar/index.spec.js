import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import NavBar from ".";

describe("NavBar", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <NavBar
          items={[
            { label: "Link" },
            { label: "Link" },
            { label: "Link", disabled: true },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
