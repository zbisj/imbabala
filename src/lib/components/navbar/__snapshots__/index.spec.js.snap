// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<ul
  className=""
>
  <li
    className=""
  >
    <a
      className=""
      href="#"
    >
      Link
    </a>
  </li>
  <li
    className=""
  >
    <a
      className=""
      href="#"
    >
      Link
    </a>
  </li>
  <li
    className=""
  >
    <a
      className=""
      disabled={true}
      href="#"
    >
      Link
    </a>
  </li>
</ul>
`;
